src/main/player/Player.java

```diff
@@ -11,6 +11,8 @@
     protected int up, left, right; // direction for player
     protected int leftBorder, rightBorder;
     public boolean ifStart;
+    public boolean ifSmash;
+    protected int smash;
```

Comment:
Not too important but you can write smash and Line11 together, cause they are same kind variable